File: app/src/main/java/com/iamoem/openglvr/MainActivity.java
package com.iamoem.openglvr;

import android.app.Activity;
import android.os.Bundle;

public class MainActivity extends Activity {

    private MyGLSurfaceView myGLSurfaceView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        myGLSurfaceView = (MyGLSurfaceView) findViewById(R.id.glSurfaceView);
    }

    @Override
    protected void onResume() {
        super.onResume();
        myGLSurfaceView.onResume();
    }
}

File: app/src/main/java/com/github/murzagalin/openglvr/MainActivity.kt
package com.github.murzagalin.openglvr

import android.app.Activity
import android.os.Bundle
import android.view.View


class MainActivity : Activity() {

    private lateinit var myGLSurfaceView: VrGlSurfaceView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        myGLSurfaceView = findViewById<View>(R.id.glSurfaceView) as VrGlSurfaceView
    }

    override fun onResume() {
        super.onResume()
        myGLSurfaceView.onResume()
    }
}
